vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -551,10 +551,12 @@ - (void)fetchGrants:(const std::string &)lang paymentId:(const std::string &)pay
   ledger->FetchGrants(lang, paymentId);
 }
 
-- (void)solveGrantCaptchWithPromotionId:(NSString *)promotionId solution:(NSString *)solution
+- (void)solveGrantCaptchWithPromotionId:(NSString *)promotionId solution:(NSString *)solution completion:(void (^)(BATResult result, ledger::GrantPtr grant))completion
 {
   ledger->SolveGrantCaptcha(std::string(solution.UTF8String),
-                            std::string(promotionId.UTF8String));
+  std::string(promotionId.UTF8String), ^(ledger::Result result, ledger::GrantPtr grant) {
+    // To be implemented
```

Comment:
At the end of this callback since we're including a completion block:

```objc++
const auto bridgedGrant = [[BATGrant alloc] initWithGrant:*grant];
completion(static_cast<BATResult>(result), bridgedGrant);
```